Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-yultsur = { git = "https://github.com/leonardoalt/yultsur", branch = "main" }
+yultsur = { git = "https://github.com/leonardoalt/yultsur", branch = "resolve_inside" }
 num-bigint = "^0.4.3"
 tempfile = "3"
 clap = "^3.2"
+num-traits = "^0.2.15"
```

Comment:
So many new deps :p

Reply:
This is already an indirect dependency of num-bigint